UPGRADE-4.0.md

```diff
@@ -24,6 +24,9 @@ Serializer
 Yaml
 ----
 
+ * The `Dumper::setIndentation()` method was removed. Pass the indentation
+   level to the constructor instead.
```

Comment:
Technically, you are right (I forgot to reply to your comment on your PR last week, sorry for that). But I think we should keep the upgrade files in sync (we did the same during the development of the 2.x versions). Otherwise we would need to spend a lot of time on syncing the files when the development of 4.0 starts.